doc/code-submission-guidelines.md

```diff
@@ -22,26 +22,35 @@ A detailed explanation of Angular's guidelines and conventions can be found [on
 Each commit message should consist of a header, a body and a footer. The header has a special format
 that includes a type, a scope and a subject. Here's a sample of the format:
 
-```html
-<type>(<scope>): <subject>
-<BLANK LINE>
-<body>
-<BLANK LINE>
-<footer>
+```
+<type>
```

Comment:
This is still incorrectly indented. We may have to opt-out of having this block auto-formatted.

Reply:
Or replace this with an example commit message (which is probably more useful anyway).

Reply:
Good spot, after making the code block (```)raw (removing html from ```html), I manually formatted the content, which obviously prettier did not change after. 

I realised by comparing to develop, my formatting style was different, so I have reapplied the formatting back to this snippet to be in sync with develop.